state/presence/presence.go

```diff
@@ -383,15 +381,20 @@ type pingInfo struct {
 }
 
 func (w *Watcher) findAllBeings() (map[int64]beingInfo, error) {
-	beings := make([]beingInfo, 0)
 	session := w.beings.Database.Session.Copy()
 	defer session.Close()
-	beingsC := w.beings.With(session)
 
-	err := beingsC.Find(bson.D{{"model-uuid", w.modelUUID}}).All(&beings)
+	beingsC := w.beings.With(session)
+	t0 := time.Now()
+	beings := make([]beingInfo, 0)
+	err := beingsC.Find(bson.M{
+		"_id": bson.M{"$regex": bson.RegEx{"^" + w.modelUUID, ""}},
+	}).All(&beings)
 	if err != nil {
 		return nil, err
 	}
+	dt := time.Now().Sub(t0)
+	logger.Tracef("[%s] loaded %d beings in %s", w.modelUUID[:6], len(beings), dt)
```

Comment:
this feels like it might be worthy for Debug. Trace is always something that "you have to know it exists to turn it on", whereas "Debug" is "would it be useful to get this sort of information from the field".